projects/forge-angular/src/lib/list/list-value-accessor.directive.ts

```diff
@@ -0,0 +1,98 @@
+import { Directive, ElementRef, forwardRef, HostListener, Renderer2, StaticProvider } from '@angular/core';
+import { ControlValueAccessor, NG_VALUE_ACCESSOR } from '@angular/forms';
+import { IListComponent, IListItemComponent, IListItemSelectEventData, LIST_ITEM_CONSTANTS } from '@tylertech/forge';
+
+export const LIST_VALUE_ACCESSOR: StaticProvider = {
+  provide: NG_VALUE_ACCESSOR,
+  useExisting: forwardRef(() => ListValueAccessor),
+  multi: true
+};
+
+@Directive({
+  selector: `forge-list[formControlName],forge-list[formControl],forge-list[ngModel]`,
+  providers: [LIST_VALUE_ACCESSOR]
+})
+export class ListValueAccessor implements ControlValueAccessor {
+  @HostListener('forge-list-item-select', ['$event'])
+  public listItemSelect(event: CustomEvent<IListItemSelectEventData>): void {
+    const { detail: { value } } =  event;
+    let listValue: any[] = this._elementRef.nativeElement.selectedValue;
+
+    //
+    // Determine control mode: Checkbox or Radio
+    //
+    const htmlTargetElement = event.target as HTMLElement;
+    const matchedInput = htmlTargetElement.querySelector(LIST_ITEM_CONSTANTS.selectors.CHECKBOX_RADIO_SELECTOR);
+    let isCheckbox = true;
+    if(matchedInput) {
```

Comment:
Not a huge deal but there should a space after each `if` for stylistic consistency